tap_zendesk/streams.py

```diff
@@ -113,7 +114,12 @@ def get_objects(self, **kwargs):
         '''
         url = self.endpoint.format(self.config['subdomain'])
 
-        for page in http.get_cursor_based(url, self.config['access_token'], **kwargs):
+        # Set and pass request timeout to config param `request_timeout` value.
+        # If value is 0,"0","" or not passed then it set default to 300 seconds.
+        config_request_timeout = self.config.get('request_timeout')
```

Comment:
@prijendev This code is already written in the `main()` function of __init__ file. What is the need of initializing these variables again? Configuration parameters should be read only once in the starting of the tap.

Reply:
In this tap some of the streams using Zenpy sdk. So, to initialize sdk client we have read request_timeout in main() function. While for other streams updated initialization of `request_timeout` to common parent Stream class to minimize the changes.